tensorflow/lite/micro/kernels/softmax_test.cc

```diff
@@ -322,4 +322,360 @@ TF_LITE_MICRO_TEST(SimpleTestQuantizedSigned) {
       output_data);
 }
 
+TF_LITE_MICRO_TEST(SimpleTestQuantizedSigned1D) {
+  using tflite::testing::F2QS;
+
+  const float input_min = -63.5f;
+  const float input_max = 64.0f;
+  const float output_min = 0.0f;
+  const float output_max = (255.0f / 256.0f);
+  const int output_dims_count = 5;
+  int8_t output_data[output_dims_count];
+  tflite::testing::TestSoftmaxQuantizedSigned(  //
+      {1, 5},                                   // Input shape.
+      {
+          F2QS(1.0, input_min, input_max),
```

Comment:
We have been trying to move away from using F2Q* in favor of CreateQuantizedTensor (see conv_test.cc).  I realize that the rest of this file uses F2Q and F2Qs, so we can leave it as-is.  Just a note for future changes.

Reply:
Acknowledged, thanks!